api/tests/acceptance/application/session/session-controller-patch-publish-session_test.js

```diff
@@ -101,18 +112,19 @@ describe('PATCH /api/admin/sessions/:id/publish', function () {
           const response = await server.inject(options);
 
           // then
-          expect(response.result.data.attributes['published-at']).to.be.an.instanceOf(Date);
-          expect(response.result.data.attributes['published-at']).to.not.equal(date);
```

Comment:
Chelou ce "to not equal" ... :thinking:

Reply:
Ca veut dire qu'il ne doit pas avoir cette valeur. 
De toute façon il est retiré

Reply:
>Ca veut dire qu'il ne doit pas avoir cette valeur.

Yes, c'est surtout l'utilisation un peu alambiquée qui en était fait que je trouvais chelou